data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/networkhandlers/PeerIDHandler.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.beaconrestapi.networkhandlers;
+
+import tech.pegasys.artemis.beaconrestapi.handlerinterfaces.BeaconRestApiHandler;
+import tech.pegasys.artemis.networking.p2p.JvmLibP2PNetwork;
+import tech.pegasys.artemis.networking.p2p.api.P2PNetwork;
+
+public class PeerIDHandler implements BeaconRestApiHandler {
```

Comment:
Should be `PeerId` with a lower-case d. Theoretically all acronyms should only hav the first letter as upper-case but it's probably the most broken style rule. :) LibP2p does have `PeerId` as a class already though so worth being consistent with that.

Reply:
Makes sense.